src/commandDetails/profile/set.ts

```diff
@@ -0,0 +1,75 @@
+import { GuildMember } from 'discord.js';
+import { CodeyUserError } from '../../codeyUserError';
+import {
+  configMaps,
+  editUserProfile,
+  UserProfile,
+  validCustomizations,
+  validCustomizationsDisplay,
+  validUserCustomization,
+} from '../../components/profile';
+import { container } from '@sapphire/framework';
+import {
+  CodeyCommandDetails,
+  CodeyCommandOptionType,
+  SapphireMessageExecuteType,
+  SapphireMessageResponse,
+} from '../../codeyCommand';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const profileSetExecuteCommand: SapphireMessageExecuteType = async (
+  client,
+  messageFromUser,
+  args,
+): Promise<SapphireMessageResponse> => {
+  const customization = <keyof typeof configMaps>args['customization'];
```

Comment:
I think "customization" should be a dropdown instead of a string field, since otherwise it's hard to tell the different options available. You can look at this for reference: https://discordjs.guide/interactions/select-menus.html#building-and-sending-select-menus

Reply:
@mcpenguin Sounds good. I'll figure this out.

Reply:
It doesn't seem like this is implemented yet.

Reply:
Implemented